### PR TITLE
K8s 1.25 release docs

### PR DESCRIPTION
**Pull request by evilnick**
## Done

- Added 1.25 section
- Update supported versions
- added release notes for 1.25
- added components page for 1.25
- added upgrade page for 1.25
- added Kube OVN docs
- Updated install instructions
- various minor fixes

NOTE: I did not add pages to navigation.yaml as usual as I see all the docs pages have been removed from there for some good reason i guess.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes

### CODE REVIEW

**Comment by evilnick** [comment]
there is going to be some press activity around this at 3pm on the 1st so would be good to get it live before then!

File: templates/kubernetes/docs/metallb.md [hunk 1]
```diff
@@ -74,32 +78,14 @@ your environment. The IP addresses can be specified as a range, such as
...
-Alternatively, you can change the config directly on the metallb-controller
-charm at any time:
+Configuring the IP addresses can be done at any time by changing the charm config via Juju.
 
 ```bash
 juju config metallb-controller iprange="192.168.1.240/28, 10.0.0.0/28"
 ```
 
 ### Note: Using RBAC
+<a name="rbac-note"> </a>
```
Comment (anthonydillon) [inline templates/kubernetes/docs/metallb.md:88]: Same here

**Review by anthonydillon — APPROVED** [review]
Generally looks good @evilnick could you review the suggestions and let me know if you want to action or ignore them?

File: templates/kubernetes/docs/1.25/components.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+layout:
+    - base
+    - ubuntu-com
+toc: false
+
+---
+
+<!-- THIS PAGE IS AUTOGENERATED -->
+<!-- If you find an error in this page, it is likely to be in the original
+source of the information - please file a bug rather than editing the text -->
```
[the remark this answers is not in the document]
Comment (evilnick) [reply templates/kubernetes/docs/1.25/components.md:22]: Hmm. I'll have to think about where the best place is for that - I will make a note to revisit though

File: templates/kubernetes/docs/1.25/components.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+-  sig-storage/csi-snapshotter:v4.2.0
+-  sig-storage/csi-snapshotter:v4.2.1
+-  sig-storage/csi-snapshotter:v5.0.1
+-  sig-storage/livenessprobe:v2.5.0
+-  sig-storage/livenessprobe:v2.6.0
+
+<!-- CONTAINER IMAGES END -->
+
+<!-- LINKS -->
+[release notes]: /kubernetes/docs/release-notes#1.25
```
[the remark this answers is not in the document]
Comment (evilnick) [reply templates/kubernetes/docs/1.25/components.md:166]: Yup!

File: templates/kubernetes/docs/1.25/components.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+[upgrade notes]: /kubernetes/docs/upgrade-notes
```
[the remark this answers is not in the document]
Comment (evilnick) [reply templates/kubernetes/docs/1.25/components.md:167]: No, that one's good - the upgrade notes lists all the stand-out issues from all versions

File: templates/kubernetes/docs/1.25/components.md [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+[bugs]: /kubernetes/docs/
+[source]: https://github.com/charmed-kubernetes
```
[the remark this answers is not in the document]
Comment (evilnick) [reply templates/kubernetes/docs/1.25/components.md:169]: :+1:

File: templates/kubernetes/docs/1.25/upgrading.md [hunk 6]
```diff
@@ -0,0 +1,437 @@
...
+kube-system                       kubernetes-dashboard-67d4c89764-nwxss                         1/1     Running   0          19m
+kube-system                       metrics-server-v0.3.1-67bb5c8d7-x9nzx                         2/2     Running   0          18m
+kube-system                       monitoring-influxdb-grafana-v4-65cc9bb8c8-mwvcm               2/2     Running   0          18m
+```
+
+<!-- LINKS -->
+
+[k8s-release]: https://github.com/kubernetes/kubernetes/releases
+[backups]: /kubernetes/docs/backups
+[release-notes]: /kubernetes/docs/release-notes
```
[the remark this answers is not in the document]
Comment (evilnick) [reply templates/kubernetes/docs/1.25/upgrading.md:420]: That one was intentional, but now I think it probably is better to point to the 1.25 version. thanks :+1:

File: templates/kubernetes/docs/1.25/components.md [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+-  sig-storage/csi-snapshotter:v5.0.1
+-  sig-storage/livenessprobe:v2.5.0
+-  sig-storage/livenessprobe:v2.6.0
+
+<!-- CONTAINER IMAGES END -->
+
+<!-- LINKS -->
+[release notes]: /kubernetes/docs/release-notes#1.25
+[upgrade notes]: /kubernetes/docs/upgrade-notes
+[bugs]: /kubernetes/docs/
```
[the remark this answers is not in the document]
Comment (evilnick) [reply templates/kubernetes/docs/1.25/components.md:168]: It isn't. it also shouldn't get used. I will check the templates, and link this to LP anyhow

**Comment by evilnick** [comment]
Thanks! I have fixed up those links. 
I haven't change the HTML anchors yet because the standard notation seems to crash the parser

``` 
## Containerd {#containerd}
```

seems to result in a Jinja error
```
jinja2.exceptions.TemplateSyntaxError: Missing end of comment tag
```

File: templates/kubernetes/docs/1.25/components.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (evilnick) [reply templates/kubernetes/docs/1.25/components.md]: I think it is easier just to point people at the kuberenets docs repo for this in hindsight (which I have done). The mechanics of the data generation are in internal jenkins jobs so it is important to report an error but not necessarily where it came from.

File: templates/kubernetes/docs/metallb.md (hunk 1 above)
Comment (evilnick) [reply templates/kubernetes/docs/metallb.md]: this is the only one that was actually referenced as far as I can tell. thanks. There are loads more scattered amongst the pages but i will make a card to tidy them up

**Comment by evilnick** [comment]
Ok, I think that's everything

**Pull request merged**